package.json

```diff
@@ -85,12 +86,12 @@
     "rimraf": "^2.5.2",
     "sanitize.css": "^3.3.0",
     "shelljs": "^0.6.0",
-    "sinon": "^1.17.3",
+    "sinon": "^2.0.0-pre",
```

Comment:
The sinon error is almost definitely due to the upgrade to a pre-release version of sinon 2.0. (ref https://github.com/sinonjs/sinon/pull/932)

Do we need to update it?